RecoHGCal/TICL/plugins/TrackstersFromSimClustersProducer.cc

```diff
@@ -104,8 +114,11 @@ void TrackstersFromSimClustersProducer::fillDescriptions(edm::ConfigurationDescr
   desc.add<edm::InputTag>("time_layerclusters", edm::InputTag("hgcalLayerClusters", "timeLayerCluster"));
   desc.add<edm::InputTag>("filtered_mask", edm::InputTag("filteredLayerClustersSimTracksters", "ticlSimTracksters"));
   desc.add<edm::InputTag>("simclusters", edm::InputTag("mix", "MergedCaloTruth"));
+  desc.add<edm::InputTag>("caloparticles", edm::InputTag("mix", "MergedCaloTruth"));
```

Comment:
You need something like [this](https://github.com/cms-sw/cmssw/blob/08fd7d924f8ef28001c9d0fec6df076fb368dce5/SimDataFormats/Associations/python/LCToCPAsssociation_cfi.py#L10-L12) to handle the premixing case.